CHANGES.md

```diff
@@ -4,6 +4,7 @@ git version
   + merlin binary
     - filter dups in source paths (#1218)
     - improve load path performance (#1323)
+    - correctly redirect stdout to stderr of ppx on Windows (#1413)
```

Comment:
I would write something a bit more general so that people will notice it if looking for the specific bug:
```suggestion
    - fix handling of ppx's under Windows (#1413)
```

Reply:
I see, thanks :) applied!